### PR TITLE
first pass at docs

### PR DESCRIPTION
**Pull request by osheroff**
@zendesk/rules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,23 @@
-# Maxwell's daemon
+<div style="float: left">
+<img alt="The Daemon, maybe" src="/img/cyberiad_1.jpg?raw=true" style="float: left; height: 300px; padding-right: 30px;">
+</div>
```
Comment (vanchi-zendesk) [inline README.md]: Suggestion: Let's create our own logo.

File: docs/docs/config.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+If maxwell finds the file "config.properties" in $PWD it will use it.
+
+option                        | description
+----------------------------- | -----------
+user=USER                     | mysql username
+password=PASSWORD             | mysql password
+host=HOST                     | mysql host
+port=PORT                     | mysql port
+producer=stdout,kafka,        | where shall we send these rows, sir?
+kafka.*=XXX                   | any options prefixed with 'kafka.' will be passed into the kafak proucer library
```
Comment (vanchi-zendesk) [inline docs/docs/config.md]: multiple typos.

File: docs/docs/config.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+
+### Kafka options
+
+Any options in the configuration file prefixed with "kafka." have that prefix stripped off, and passed directly
+into the "new producer" configuration, as desribed here: [http://kafka.apache.org/documentation.html#newproducerconfigs](http://kafka.apache.org/documentation.html#newproducerconfigs)
```
Comment (vanchi-zendesk) [inline docs/docs/config.md]: typo. Do  a spellcheck. :D

**Comment by vanchi-zendesk** [comment]
:+1: for the documentation. Strongly suggest we pay for our own logo and images.

**Comment by osheroff** [comment]
concern noted, ignored.

**Pull request merged**